### DISCUSSION TITLE
paddleocr 在华为910b的npu上跑很慢且只跑了一张卡

### DISCUSSION THREAD

**Discussion by danyXu** [PaddlePaddle/PaddleOCR#12768] [Q&A]
#### 问题描述 / Problem Description
paddleocr 在华为910b上运行很慢且只跑了一张卡，总共8张

#### 运行环境 / Runtime Environment
-paddle-custom-npu 0.0.0
paddle2onnx 1.0.5
paddleclas 2.5.2
paddleocr 2.7.3
paddlepaddle 3.0.0.dev20240527

#### 复现代码 / Reproduction Code
paddleocr --image_dir imgs/123fail_0.png

#### 完整报错 / Complete Error Message
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/3701158/5844f943-3af0-4d56-ad61-1865c46eb0d3)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/3701158/adf2e8d6-711b-4501-92f8-1960211ec9d0)

#### 可能解决方案 / Possible solutions
无参数可设置多卡运行，npu跑的很慢也无参数可调

**Comment by danyXu** [comment]
cpu运行就很快  
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/3701158/e0fcbf8c-109f-4c88-b8d1-7aa374dfcca0)

**Comment by GreatV** [comment]
可以在这问问看

https://github.com/PaddlePaddle/PaddleCustomDevice/issues
